packages/mui-base/src/ClickAwayListener/ClickAwayListener.tsx

```diff
@@ -22,7 +22,12 @@ function clickedRootScrollbar(event: MouseEvent, doc: Document) {
   );
 }
 
-type ClickAwayMouseEventHandler = 'onClick' | 'onMouseDown' | 'onMouseUp';
+type ClickAwayMouseEventHandler =
```

Comment:
We should also update the [`mapEventPropToEvent`](https://github.com/mui/material-ui/blob/37fe36b7468e76be4af009dfa9ad0a6633804ae8/packages/mui-base/src/ClickAwayListener/ClickAwayListener.tsx#L12) too.

Reply:
Good catch! 🥇